lib/routes/account.js

```diff
@@ -630,6 +630,46 @@ module.exports = function (
       }
     },
     {
+      method: 'POST',
+      path: '/account/status',
+      config: {
+        validate: {
+          payload: {
+            email: validators.email().required()
+          }
+        },
+        response: {
+          schema: {
+            exists: isA.boolean().required()
+          }
+        }
+      },
+      handler: function (request, reply) {
+        var email = request.payload.email
+
+        customs.check(
+          request.app.clientAddress,
+          email,
+          'accountStatusCheck')
+          .then(
+            db.accountExists.bind(db, email)
+          )
+          .done(
+            function (exist) {
+              reply({
+                exists: exist
+              })
+            },
+            function (err) {
+              if (err.errno === error.ERRNO.ACCOUNT_UNKNOWN) {
+                return reply({ exists: false })
+              }
+              reply(err)
+            }
```

Comment:
I don't believe `db.accountExists` will ever throw `ACCOUNT_UNKNOWN`, but no harm in having it here.  I suspect we'll continue to iterate this API to add more info about the account status.